app/src/main/res/xml/account_settings_preferences.xml

```diff
@@ -1,54 +1,54 @@
-<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-
 <PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <androidx.preference.Preference
-        app:iconSpaceReserved="false"
+    <Preference
         android:key="@string/pref_key_sync_now"
-        android:title="@string/preferences_sync_now" />
+        android:title="@string/preferences_sync_now"
+        app:icon="@drawable/mozac_ic_sync"
+        android:layout="@layout/account_preference_description_icon" />
 
-    <androidx.preference.EditTextPreference
-        app:iconSpaceReserved="false"
+    <EditTextPreference
         android:key="@string/pref_key_sync_device_name"
+        android:layout="@layout/account_preference_description_icon"
```

Comment:
I've also added this because 3dp margin between title and subtitle both for device name and sync now.

Reply:
![resim](https://user-images.githubusercontent.com/17825767/86160542-93b3de00-bb14-11ea-846b-e2c5861d3b55.png)

Reply:
@NotWoods

Reply:
For very small adjustments like this, I prefer to defer to what Android does natively rather than introducing custom layouts.

Reply:
Thanks removing layout.